observability/logging/logging-6.2/log6x-opentelemetry-data-model-6.2.adoc

```diff
@@ -0,0 +1,383 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-opentelemetry-data-model-6-2"]
+= OpenTelemetry data model
+include::_attributes/common-attributes.adoc[]
+:context: log6x-opentelemetry-data-model-6-2
+
```

Comment:
@JoaoBraveCoding @xperimental @theashiot 
 This doc confuses me, as i can not see the fields body,observedTimeUnixNano, timeUnixNano, severityText,attributes in lokistack.  I think these fields are intermediate fields using in data transfer.  the customer shouldn't be aware of this. This doc should appear in development  phase, not here. 
 We only need to explain how to enable Otel  in CLF and what is the expected records in lokistack. I suggest rewriting this doc using the same structure as syslog, httpserver.

Reply:
We can rewrite this chapter following step below.

1.    Explain what is OpenTelemetry data model in simple words.
2.    how to enable OTEL in CLF
-  use otel output send logs to lokistack or the other otel compatible receiver(for example otel collector).
-  use lokistack output otel mode
3.   The records examples

Reply:
From my POV it's hard to comment on this one, but I believe this document was written precisely to give insight on how Logging is doing OTLP and to that end I feel the document accomplishes that but I'll defer this one to @xperimental

Reply:
If we can, I think I would just link to the original source of this information ([this](https://github.com/rhobs/observability-data-model/blob/main/cluster-logging.md) file) as "reference material". This avoids the pitfall that we need to keep to copies updated and as Anping already mentioned this is more "internals" anyway.

Reply:
My preference would be to rely upon the upstream doc as a source of truth.  I don't know what our policy is but  it would ensure there is no drift.

Reply:
How about we publish this for now, as this was a part of 6.1: https://docs.openshift.com/container-platform/4.17/observability/logging/logging-6.1/log6x-opentelemetry-data-model-6.1.html